cylc/flow/scripts/install.py

```diff
@@ -27,13 +27,13 @@
 
 If a SOURCE_NAME is supplied, Cylc will search for the workflow source in the
 list of directories given by "global.cylc[install]source dirs", and install
-the first match. The installed workflow name will be the same as SOURCE_NAME,
+the first match. The installed workflow ID will be the same as SOURCE_NAME,
```

Comment:
Not this one I think

```suggestion
the first match. The installed workflow name will be the same as SOURCE_NAME,
```
or perhaps
```suggestion
the first match. The installed workflow ID will start with SOURCE_NAME,
```